crossdock/otel-agent-config.yml

```diff
@@ -4,15 +4,19 @@ receivers:
       host_endpoint: localhost:5778
       fetch_endpoint: jaeger-collector:14250
     protocols:
+      grpc:
```

Comment:
Why has this been added (and disabled) rather than just leaving out?

Reply:
Because grpc is enabled by default and our legacy agent doesn't expose gRPC endpoint. Once we provide agent build with opinionated config this will be removed.